bricksrc/version.py

```diff
@@ -0,0 +1,17 @@
+# Major version: increment for backwards compatible changes
+BRICK_MAJOR_VERSION = 1
+
+# Minor version: increment for substantial additions to the ontology that
+# are backwards compatible
+BRICK_MINOR_VERSION = 1
+
+# Patch version: increment for minor additions/changes to the ontology that
+# are largely backwards compatible (bug-fixes exempted)
```

Comment:
What does "largely backward compatible" mean? It reads like PATCH_VERSION has a looser rule than MINOR_VERSION does.